.github/workflows/ci.yaml

```diff
@@ -0,0 +1,133 @@
+---
+name: "CI"
+on:
+  push:
+    branches: ["develop"]
+  pull_request:
+    branches: ["develop"]
```

Comment:
Does this mean it won't run for a PR against, say, `develop-1.1.0`?

Reply:
That is correct for now... We probably need to update this to `["main","develop","develop-1.1.0"]` correct?  Any others?

Reply:
That's probably good for now - is there any wildcarding capability? It'd be nice to be able to use `"develop.*"` or similar so that we don't have to update this file for `develop-1.2.0` etc as new integration branches are created.